cla_public/templates/base.html

```diff
@@ -105,7 +105,8 @@
     {% if zeroOptions %}
       // this is set in diagnosis.html
       $( document ).ready(function(){
-        ga('moj.send', 'event', 'Zero-options', 'Zero options instance', 'No options instance: '+stripPII(window.location.href));
+        ga('moj.send', 'event', 'Zero-options', 'Zero options (page reload)', 'No options instance: '+stripPII(window.location.href));
+        location.reload();
```

Comment:
This could maybe lead them to getting no options again and be stuck on this page.
Can we redirect them to the start page instead?

Reply:
Every instance of this is solved by refreshing the page.  I've never had the same error occur twice in succession.  However, if that were to happen, it would just reload and then be fine.  

Going to the start page would end their session, so that's not good...